test/e2e/node/pod_resize.go

```diff
@@ -26,29 +26,26 @@ import (
 	"strings"
 	"time"
 
+	semver "github.com/blang/semver/v4"
+	"github.com/google/go-cmp/cmp"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
```

Comment:
nit: there should be 3 import blocks:
1. standard library
2. 3rd party imports
3. Kubernetes imports (anything under k8s.io)